Test/GherkinFeatureFiles/display-points.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Data Point Component in WebXR Application
+  Background:
+    Given That the user is on the Meta Quest 2
```

Comment:
Can we move the WebXR application is loaded into the background?
something like
  Background:
    Given that the user is on a Meta Quest 2
    And they are inside of the quest browser
    And they have loaded the web app